tests/src/Mod/Material/App/TestMaterialCards.cpp

```diff
@@ -78,11 +82,11 @@ TEST_F(TestMaterialCards, TestCopy)
     EXPECT_TRUE(_library);
```

Comment:
Another side note: I typically use `ASSERT_TRUE` with my tests' preconditions, so that if the precondition isn't met, the test doesn't run at all and fails immediately.